lib/omnibus/fetchers/git_fetcher.rb

```diff
@@ -165,7 +171,9 @@ def git_checkout
     def git_fetch
       fetch_cmd = "fetch #{source_url} #{described_version}"
       fetch_cmd << " --recurse-submodules=on-demand" if clone_submodules?
-      git(fetch_cmd)
+      retry_block("git fetch", [CommandTimeout, CommandFailed]) do
```

Comment:
@schisamo I can certainly add more info, the only reason I didn't is because if it fails the retry, then you get the full pretty-printing of the exception which has a lot of info.  This is just for the log message telling you it was retried.  I am happy with adding more info though if you think it'll be helpful.